### PR TITLE
Silence java warnings after import.

### PR DESCRIPTION
**Pull request by thmsbinder**

### CODE REVIEW

**Review by tiziano88 — APPROVED** [review]
Is the linter / formatter set up in a way that will keep the current format? Or is that not possible?

**Comment by thmsbinder** [comment]
> Is the linter / formatter set up in a way that will keep the current format? Or is that not possible?

Yes, that must be possible. It is the bare minimum.

FWIW build_cleaner does not exist outside Google, for technical reasons only - it seems difficult even for java.

**Comment by tiziano88** [comment]
Let me rephrase: just considering formatting for now, is there something *now* that enforces it? I don't think so, since both the previous and the current format were accepted by CI. If so, what tool should be used to keep the code formatted?

**Review by ipetr0v — APPROVED** [review]

**Comment by thmsbinder** [comment]
> Let me rephrase: just considering formatting for now, is there something _now_ that enforces it? I don't think so, since both the previous and the current format were accepted by CI. If so, what tool should be used to keep the code formatted?

We should make the Java Language Support by Red Hat extension mandatory (assuming everyone uses VS Code). Then use the Java formatter with particular settings. Settings come out at oak/.vscode/java-formatter.xml. Is this something we could/should add to GitHub?

**Comment by tiziano88** [comment]
Sure we can look into VScode at some point too, but mainly I'm talking about CI, and specifically ensuring that Java files are formatted (probably adding something around here: https://github.com/project-oak/oak/blob/c7027badbd0d6dbb2256f25fa21740b31cc43781/xtask/src/main.rs#L216-L220 )

**Comment by tiziano88** [comment]
In short, what is the actual binary that needs to be run in order to format the java files in the same way as google3?

**Comment by tiziano88** [comment]
It turns out that we already have clang-format which is supposed to format java files, but apparently that's not working: https://github.com/project-oak/oak/blob/c7027badbd0d6dbb2256f25fa21740b31cc43781/xtask/src/files.rs#L117

Could you take a look?
